layouts/partials/scripts.html

```diff
@@ -18,7 +18,13 @@
 </style>
 <script>
 window.markmap = {
-  autoLoader: { manual: true },
+  autoLoader: { 
+      manual: true,
+      onReady() {
+        const { autoLoader, builtInPlugins } = window.markmap;
+        autoLoader.transformPlugins = builtInPlugins.filter(plugin => plugin.name !== 'prism');
+      },
+  },
```

Comment:
Hmm, I'm wondering if this should have been added to https://github.com/google/docsy/blob/main/assets/js/markmap.js?

Reply:
Created #1188 to track this.